.travis.yml

```diff
@@ -22,3 +21,11 @@ script: python -m pytest --cov=nf_core .
 
 after_success:
   - codecov
+
+deploy:
+  provider: pypi
+  user: ewels
```

Comment:
According to https://docs.travis-ci.com/user/deployment/pypi/, the user and password needs to be surrounded with quotation marks, no idea if this will cause a failure. I have it with quotation marks for my project, and it works.

Reply:
Agreed, that's safer 👍